pkg/scaffold/controller/controller.go

```diff
@@ -87,6 +81,24 @@ func (a *Controller) GetInput() (input.Input, error) {
 	return a.Input, nil
 }
 
+func (a *Controller) bindResource(coreGroups map[string]string) {
+	resourcePath := filepath.Join("pkg", "apis", a.Resource.Group, a.Resource.Version,
+		fmt.Sprintf("%s_types.go", strings.ToLower(a.Resource.Kind)))
+	if _, err := os.Stat(resourcePath); os.IsNotExist(err) {
+		if domain, found := coreGroups[a.Resource.Group]; found {
+			a.ResourcePackage = path.Join("k8s.io", "api")
+			a.GroupDomain = a.Resource.Group
+			if domain != "" {
+				a.GroupDomain = a.Resource.Group + "." + domain
+			}
+			return
+		}
+		// TODO: need to support '--resource-pkg-path' flag for specifying resourcePath
+	}
+	a.ResourcePackage = path.Join(a.Repo, "pkg", "apis")
+	a.GroupDomain = a.Resource.Group + "." + a.Domain
+}
```

Comment:
we can capture this logic in a separate function so that it can be called by both Test/Controller methods ?

Reply:
Done